src/wallets/ethereum.ts

```diff
@@ -137,7 +141,7 @@ export class Web3EthereumWallet implements EthereumWallet {
     const serializedTx = tx.serialize();
     const hex = "0x" + serializedTx.toString("hex");
 
-    log(`Invoking web3.eth.sendSignedTransaction with ${hex}`);
+    logger.debug(`Invoking web3.eth.sendSignedTransaction with ${hex}`);
```

Comment:
🙃 I would see this as `trace` as it tells you what's currently happening. 
trace includes debug, so this is just a personal preference. 

```suggestion
    logger.trace(`Invoking web3.eth.sendSignedTransaction with ${hex}`);
```

Reply:
it goes a bit beyond `trace` with the level of information, and is helpful for debugging.